lib/visualization.js

```diff
@@ -63,6 +63,10 @@ class Visualization {
 		this.pause();
 		d3.select('#vis_svg').remove();
 	}
+
+	action_map(){ //maps action to envt specific move
```

Comment:
oh, let's use this. it'll look good. you can find the mapping in Gridworld.actions